genai-perf/genai_perf/inputs/converters/openai_embeddings_converter.py

```diff
@@ -31,33 +31,25 @@
 
 
 class OpenAIEmbeddingsConverter(BaseConverter):
-    def convert(
-        self,
-        generic_dataset: Dict,
-        config: InputsConfig,
-    ) -> Dict:
-        pa_json: Dict[str, Any] = {"data": []}
 
-        for index, entry in enumerate(generic_dataset["rows"]):
-            iter_model_name = self._select_model_name(config, index)
-            payload = entry.get("payload", {})
-            input_values = payload.get("input")
+    def convert(self, generic_dataset: Dict, config: InputsConfig) -> Dict:
+        """
```

Comment:
Do we want to have these sorts of function descriptions in the base converter instead of copying them into each converter? Open either way on this, with a slight preference for just having the shared function descriptions in the base class.

Reply:
removed the docstrings for the derived classes